src/helpers/proxyRequest.ts

```diff
@@ -4,24 +4,26 @@ import Logger from "./logger";
 
 export function proxyRequest(ipfsBackend: string) {
   const logger = new Logger("ProxyRequest");
-  return async function (req: Express.Request, res: Express.Response) {
+  return function (req: Express.Request, res: Express.Response) {
     logger.info(`Request received for ${req.url}`);
 
     logger.debug(`Request body received`);
-    try {
-      const proxyResponse = await axios({
-        method: req.method,
-        url: `${ipfsBackend}${req.originalUrl}`,
-        headers: req.headers as any,
-        data: req.body,
+
+    axios({
+      method: req.method,
+      url: `${ipfsBackend}${req.originalUrl}`,
+      headers: req.headers as any,
```

Comment:
Not part the PR's scope, but do you think that the proxy should shield the API keys sent to the IPFS cluster?
I would suggest having a set of API keys for the public facing endpoints and a specific private API key used to interact with the IPFS cluster.

Reply:
IPFS cluster doesn't support api keys
that is one of the reasons why we need this proxy..........